packages/falter-berlin-freifunk-defaults/uci-defaults/freifunk-berlin-freifunk-defaults

```diff
@@ -21,3 +21,7 @@ uci set freifunk.olsr_interface.Ip4Broadcast='255.255.255.255'
 uci set freifunk.dhcp.leasetime='5m'
 
 uci commit freifunk
+
+uci set uhttpd.main.redirect_https=1
+uci commit uhttpd
+service uhttpd reload
```

Comment:
please put this in falter-berlin-uhttpd-defaults

Reply:
resolved in recent version.